ironfish/src/primitives/rawTransaction.ts

```diff
@@ -17,6 +22,8 @@ import { Transaction } from './transaction'
 // Needed for constructing a witness when creating transactions
 const noteHasher = new NoteHasher()
 
+export const TYPE_NUMBER_SIZE = 8
```

Comment:
I think we should omit this from this PR since it focuses on hard-coded values for the transaction

Reply:
Bump on this, I don't think we need this and I don't think it should be defined in the raw transaction file